apps/src/lib/node/ledger/ethereum_node/oracle/config.rs

```diff
@@ -0,0 +1,26 @@
+//! Configuration for an oracle.
+use namada::types::ethereum_events::EthAddress;
+
+/// Configuration for an oracle.
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Hash, Ord, PartialOrd)]
+pub struct Config {
+    /// The minimum number of block confirmations an Ethereum block must have
+    /// before it will be checked for bridge events.
+    pub min_confirmations: u64,
```

Comment:
```suggestion
    pub min_confirmations: NonZeroU64,
```
otherwise, we accept forged events that aren't even recorded on an ethereum block by setting `min_confirmations = 0`